.coveragerc

```diff
@@ -1,49 +1,35 @@
-[run]
-source = tardis
-
 [report]
+# Regexes for lines to exclude from consideration
 exclude_lines =
-   # Have to re-enable the standard pragma
-   pragma: no cover
-
-   # Don't complain about packages we have installed
-   except ImportError
+    # Have to re-enable the standard pragma
+    pragma: no cover
 
-   # Don't complain if tests don't hit assertions
-   raise AssertionError
-   raise NotImplementedError
+    # Don't complain about missing debug-only code:
+    def __repr__
+    if self\.debug
 
-   # Don't complain about script hooks
-   def main\(.*\):
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AssertionError
+    raise NotImplementedError
 
-   # Ignore branches that don't pertain to this version of Python
-   pragma: py{ignore_python_version}
+    # Don't complain if non-runnable code isn't run:
+    if 0:
+    if __name__ == .__main__.:
 
 omit =
+    *__init__*
+    *_astropy_init*
+    *version*
+    *tests*
+    *conftest*
+    *setup_package*
+
+    tardis/analysis.py
+    tardis/gui/*
```

Comment:
I assume this is the deprecated GUI

Reply:
Yes it is. I don't know who made the decision of leaving the GUI outside the coverage, because it has some tests. But I think visualization team was planning to deprecate it soon.

Reply:
Also, Is a good decision to leave out the `scripts` folder? IIRC `cmfgen2tardis.py` wasn't working correctly.

Reply:
Perhaps not, even if something is not working it should probably be checked for test coverage!

Reply:
Ok, I removed it. Do you know why analysis and stats are out of coverage?

Reply:
analysis should definitely be in coverage, stats seems to be a deprecated or otherwise unfinished part

Reply:
Ok, solved!